cgi/product_multilingual.pl

```diff
@@ -964,302 +923,47 @@ ($$)
-<ul id="manage_images_accordion" class="accordion" data-accordion>
-  <li class="accordion-navigation">
-<a href="#manage_images_drop">@{[ display_icon('collections') ]} $Lang{manage_images}{$lc}</a>
-
-
-<div id="manage_images_drop" class="content" style="background:#eeeeee">
-
-HTML
-. display_select_manage($product_ref) .
-<<HTML
-
-	<p>$Lang{manage_images_info}{$lc}</p>
-	<a id="delete_images" class="button small disabled">@{[ display_icon('delete') ]} $Lang{delete_the_images}{$lc}</a><br/>
-	<div class="row">
-		<div class="small-12 medium-5 columns">
-			<button id="move_images" class="button small disabled">@{[ display_icon('arrow_right_alt') ]} $Lang{move_images_to_another_product}{$lc}</a>
-		</div>
-		<div class="small-4 medium-2 columns">
-			<label for="move_to" class="right inline">$Lang{barcode}{$lc}</label>
-		</div>
-		<div class="small-8 medium-5 columns">
-			<input type="text" id="move_to" name="move_to" />
-		</div>
-	</div>
-	<input type="checkbox" id="copy_data" name="copy_data"><label for="copy_data">$Lang{copy_data}{$lc}</label>
-	<div id="moveimagesmsg"></div>
-</div>
-</li>
-</ul>
-
-HTML
-;
-
-	$styles .= <<CSS
-.show_for_manage_images {
-line-height:normal;
-font-weight:normal;
-font-size:0.8rem;
-}
-
-.select_manage .ui-selectable li { height: 180px }
-
-CSS
-;
-
-
-	$initjs .= <<JS
-
-\$('#manage_images_accordion').on('toggled', function (event, accordion) {
-
-	toggle_manage_images_buttons();
-});
-
-
-
-\$("#delete_images").click({},function(event) {
-
-event.stopPropagation();
-event.preventDefault();
-
-
-if (! \$("#delete_images").hasClass("disabled")) {
-
-	\$("#delete_images").addClass("disabled");
-	\$("#move_images").addClass("disabled");
-
- \$('div[id="moveimagesmsg"]').html('<img src="/images/misc/loading2.gif" /> ' + lang().product_js_deleting_images);
- \$('div[id="moveimagesmsg"]').show();
-
-	var imgids = '';
-	var i = 0;
-	\$( "#manage .ui-selected"  ).each(function() {
-		var imgid = \$( this ).attr('id');
-		imgid = imgid.replace("manage_","");
-		imgids += imgid + ',';
-		i++;
-});
-	if (i) {
-		// remove trailing comma
-		imgids = imgids.substring(0, imgids.length - 1);
-	}
-
- \$("#product_form").ajaxSubmit({
-
-  url: "/cgi/product_image_move.pl",
-  data: { code: code, move_to_override: "trash", imgids : imgids },
-  dataType: 'json',
-  beforeSubmit: function(a,f,o) {
-  },
-  success: function(data) {
-
-	if (data.error) {
-		\$('div[id="moveimagesmsg"]').html(lang().product_js_images_delete_error + ' - ' + data.error);
-	}
-	else {
-		\$('div[id="moveimagesmsg"]').html(lang().product_js_images_deleted);
-	}
-	\$([]).selectcrop('init_images',data.images);
-	\$(".select_crop").selectcrop('show');
-
-  },
-  error : function(jqXHR, textStatus, errorThrown) {
-	\$('div[id="moveimagesmsg"]').html(lang().product_js_images_delete_error + ' - ' + textStatus);
-  },
-  complete: function(XMLHttpRequest, textStatus) {
-
+		$template_data_ref_display->{copy_data} = $Lang{copy_data}{$lc};
```

Comment:
we can remove those lines and use directly the lang() function in the template

Reply:
using the lang function after removing these lines results in hash number instead of text. 
![Screenshot from 2021-07-19 12-44-41](https://user-images.githubusercontent.com/47316899/126148562-2150b7ec-e57d-4c97-9c3f-3ef3ae7f790a.png)